.github/workflows/e2e-ci.yaml

```diff
@@ -9,24 +9,27 @@ on: # rebuild any PRs and main branch changes
     inputs:
       RunE2ETestsOnly:
         description: 'Run E2E tests only'
-        default: false
+        default: 'false'
+        required: false
       RunLoadTests:
         description: 'Include Load Tests in current run'
         default: 'false'
+        required: false
       TestsToRun:
         default: '[SensorDecodingTest,OTAAJoinTest,ABPTest,OTAATest,MacTest,ClassCTest,C2DMessageTest,MultiGatewayTest,MultiConcentratorTest,CupsTest,LnsDiscoveryTest,CloudDeploymentTest]'
         description: 'tests to run'
+        required: false
       TxPower:
         description: 'TXPower value to use in E2E tests'
         default: '6'
+        required: false
+permissions:
```

Comment:
Why is this needed?

Reply:
It is needed for OIDC authentication https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers#adding-permissions-settings